foursight_core/react/api/envs.py

```diff
@@ -33,23 +33,25 @@ def get_known_envs(self) -> list:
     def get_known_envs_count(self) -> int:
         return len(self._known_envs)
 
-    @memoize(100)
+    @lru_cache(100)
```

Comment:
I wonder if what you really wanted to do was (and this is definitely untested, but I'm pretty sure will work):
```
memoize = lru_cache(100)  # Create a reusable decorator based on standard parameters.

...
@memoize
def ...whatever...(...):
    ...
```

Reply:
Yeah I like it. Thanks. Updating.